app/src/main/java/com/github/andrezimmermann/gluappsample/server/converter/JsonConverter.java

```diff
@@ -0,0 +1,110 @@
+package com.github.andrezimmermann.gluappsample.server.converter;
+
+
+import com.github.andrezimmermann.gluappsample.server.data.ConvertableData;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.TypeAdapter;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonToken;
+import com.google.gson.stream.JsonWriter;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class JsonConverter implements DataConverter {
+
+    Gson gson = getGson();
+
+    private Gson getGson() {
+
+        //Java 1.6 and below can use default pattern
+        String property = System.getProperty("java.specification.version");
+        double javaVersion = Double.parseDouble(property);
+
+
+        if (javaVersion <= 1.6) {
```

Comment:
Just out of curiosity, how did you find this problem with the java platform version?